### PR TITLE
Welcome section

### PR DESCRIPTION
**Pull request by phann-phearoen**
Welcome section added, and previous components are now working across all devices size.

### CODE REVIEW

File: src/components/Douga.vue [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        else
+            this.screenMode = 'mobile';
+        }
+    }
+}
+</script>
+
+<style scoped>
+
+</style>TabletMobile
```
Comment (geonsangyoo) [inline src/components/Douga.vue:49]: @phann-phearoen 
Please add a newline('\n') at the end of last lline(:49)

File: src/components/Top.vue [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    position: relative;
+    display: block;
+}
+.plus-image {
+    left: 30%;
+    top: 30%;
+    position: absolute;
+    width: 40%;
+}
+</style>
```
Comment (geonsangyoo) [inline src/components/Top.vue:20]: @phann-phearoen 
Please add a newline('\n') at the end of last lline(:20)

File: src/components/douga-toggles/Mobile.vue [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    left: 10%;
+    top: 20%;
+    position: absolute;
+    width: 80%;
+    font-size: .7em;
+    color: grey;
+    font-family: 'M PLUS Rounded 1c', sans-serif;
+    font-weight: 400;
+}
+</style>
```
Comment (geonsangyoo) [inline src/components/douga-toggles/Mobile.vue:49]: @phann-phearoen 
Please add a newline('\n') at the end of last lline(:49)

File: src/components/douga-toggles/Normal.vue [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    left: 19%;
+    top: 30%;
+    position: absolute;
+    width: 60%;
+    font-size: 1em;
+    color: grey;
+    font-family: 'M PLUS Rounded 1c', sans-serif;
+    font-weight: 600;
+}
+</style>
```
Comment (geonsangyoo) [inline src/components/douga-toggles/Normal.vue:83]: @phann-phearoen 
Please add a newline('\n') at the end of last lline(:83)

File: src/components/douga-toggles/Tablet.vue [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    left: 12%;
+    top: 40%;
+    position: absolute;
+    width: 75%;
+    font-size: 1rem;
+    color: grey;
+    font-family: 'M PLUS Rounded 1c', sans-serif;
+    font-weight: 500;
+}
+</style>
```
Comment (geonsangyoo) [inline src/components/douga-toggles/Tablet.vue:49]: @phann-phearoen 
Please add a newline('\n') at the end of last lline(:49)

File: src/store/dougaModule.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+                clip: '../assets/clip.png',
+                description: `テキストテキストテキストテキストテキストテキストテキストテキストテキストテキストテキストテキスト。
+                    テキストテキストテキストテキストテキストテキストテキストテキストテキストテキストテキストテキスト。`
+            },
+        }
+    },
+    
+});
+
+export default douga;
```
Comment (geonsangyoo) [inline src/store/dougaModule.js:18]: @phann-phearoen 
Please add a newline('\n') at the end of last lline(:18)

File: src/store/index.js [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+        }
+    },
+    getters: {
+        welcomeTexts (state) {
+            return state.welcomeTexts;
+        }
+    }
+});
+
+export default store;
```
Comment (geonsangyoo) [inline src/store/index.js:25]: @phann-phearoen 
Please add a newline('\n') at the end of last lline(:25)

File: src/components/Douga.vue [hunk 8]
```diff
@@ -0,0 +1,49 @@
+<template>
+    <normal :welcomeTexts="welcomeTexts" v-if="screenMode === 'normal'"></normal>
+    <tablet :welcomeTexts="welcomeTexts" v-else-if="screenMode === 'tablet'"></tablet>
+    <mobile :welcomeTexts="welcomeTexts" v-else></mobile>
```
Comment (geonsangyoo) [inline src/components/Douga.vue:4]: @phann-phearoen 

Please refer to the feedbacks on your PR(#2) .
[Vuetify Grid System] is recommended for both resizing and device responsiveness simultaneously.

**Review by geonsangyoo — CHANGES_REQUESTED** [review]

File: src/components/Douga.vue (hunk 8 above)
Comment (phann-phearoen) [reply src/components/Douga.vue:4]: There's a problem with that because Vutify is not compatible with Vue3 yet! It said to be released later this year.

**Pull request merged**